static/js/stream_popover.js

```diff
@@ -500,6 +515,41 @@ exports.register_topic_handlers = function () {
 
         e.stopPropagation();
     });
+
+    $('body').on('click', '.sidebar-popover-move-topic-messages', function (e) {
+        const topic_row = $(e.currentTarget);
+        const stream_id = topic_row.attr('data-stream-id');
```

Comment:
Our pattern is 
`        const stream_id = parseInt(topic_row.attr('data-stream-id'), 10);          `


(And not to do `+stream_id`, below).